### ISSUE TITLE
.NET 5 not working with Helixtoolkit

### ISSUE THREAD

**Issue by bsohaliya** [helix-toolkit/helix-toolkit#1720]
I have changed my WPF-project from .net Framework 4.5 to .NET 5. I have added the nuget-package **Helixtoolkit.Core.Wpf** from Helixtoolkit.
I built and run the app and now the App is not showing any models except boundaries, Frame-rate, coordinate-system and ViewCube.
The whole project run perfectly with .net Framework. Which things am I missing? 
Here is the XAML code:

<Controls:MetroWindow
    x:Class="SyNETViewer.MainWindow"
    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
    xmlns:Controls="http://metro.mahapps.com/winfx/xaml/controls"
    xmlns:Helix="http://helix-toolkit.org/wpf"
    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
    xmlns:intr="http://schemas.microsoft.com/xaml/behaviors"
    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
    Title="SyNET Viewer"
    Icon="robotView.ico"
    WindowStartupLocation="CenterScreen"
    mc:Ignorable="d">

    <intr:Interaction.Triggers>
        <intr:EventTrigger EventName="Loaded">
            <intr:InvokeCommandAction Command="{Binding WindowLoadedCommand}" />
        </intr:EventTrigger>
    </intr:Interaction.Triggers>

    <Border Padding="10">
        <Grid>
            <Grid.ColumnDefinitions>
                <ColumnDefinition Width="*" />
            </Grid.ColumnDefinitions>
            <Grid>
                
                <Helix:HelixViewport3D
                    Name="m_helix_viewport"
                    Margin="0,0,0,0"
                    Background="#32373C"
                    BorderBrush="{StaticResource MahApps.Brushes.Accent}"
                    BorderThickness="5"
                    ClipToBounds="False"
                    CoordinateSystemHeight="150"
                    CoordinateSystemHorizontalPosition="Right"
                    CoordinateSystemLabelForeground="Black"
                    CoordinateSystemVerticalPosition="Center"
                    CoordinateSystemWidth="150"
                    IsManipulationEnabled="True"
                    IsViewCubeEdgeClicksEnabled="True"
                    RenderOptions.BitmapScalingMode="HighQuality"
                    RenderOptions.EdgeMode="Unspecified"
                    ShowCameraInfo="False"
                    ShowCoordinateSystem="True"
                    ShowFrameRate="True"
                    ShowViewCube="True"
                    ViewCubeHeight="150"
                    ViewCubeVerticalPosition="Bottom"
                    ViewCubeWidth="150"
                    ZoomExtentsWhenLoaded="False">

                    
                    <Helix:SunLight />

                    <ModelVisual3D Content="{Binding SyNETAnimationViewModel3D.SyNETModel3D}" />

                    <Helix:HelixViewport3D.Camera>
                        <OrthographicCamera
                            Width="2500"
                            LookDirection="-1691.75, 0.0, -1691.75"
                            Position="1739.75, -32, 1770.75"
                            UpDirection="0, 0, 1" />
                    </Helix:HelixViewport3D.Camera>
                </Helix:HelixViewport3D>

                <Grid
                    Margin="10"
                    HorizontalAlignment="Left"
                    VerticalAlignment="Bottom"
                    Background="White"
                    Opacity="0.5">
                    <Grid.ColumnDefinitions>
                        <ColumnDefinition />
                        <ColumnDefinition />
                    </Grid.ColumnDefinitions>
                    <Grid Grid.Column="0">
                        <TextBlock
                            Margin="5"
                            Foreground="Black"
                            Text="{Binding SyNETAnimationViewModel3D.DobotVM.m_currentPose}"
                            TextBlock.FontWeight="Bold" />
                    </Grid>
                    <Grid Grid.Column="1">
                        <TextBlock
                            Margin="0,5,5,5"
                            Foreground="Black"
                            Text="{Binding SyNETAnimationViewModel3D.DobotVM.m_currentJoint}"
                            TextBlock.FontWeight="Bold" />
                    </Grid>
                </Grid>

            </Grid>
        </Grid>
    </Border>
</Controls:MetroWindow>

**Comment by bsohaliya** [comment]
> Does .net 4.5 works?

The whole project run perfectly with .net 4.7 Framework.

**Comment by holance** [comment]
Probably you can still use HelixToolkit.WPF package instead of HelixToolkit.Core.Wpf? HelixToolkit.Core.Wpf is used for dot net core.

**Comment by bsohaliya** [comment]
> Probably you can still use HelixToolkit.WPF package instead of HelixToolkit.Core.Wpf? HelixToolkit.Core.Wpf is used for dot net core.

The HelixToolkit.WPF package is not compatible with .NET 5. Because .NET 5 is the next release of .NET Core following 3.1. Therefore I have installed HelixToolkit.Core.Wpf package to my project.

**Comment by holance** [comment]
Hmm, probably you should check the binding. We didn't change any code when switching to dot net core wpf.

**Comment by bsohaliya** [comment]
> Hmm, probably you should check the bindings. We didn't change any code when switching to dot net core wpf. Which version of the Helixtoolkit.Core.Wpf are you using right now?

I am using latest version of the Helixtoolkit.Core.Wpf (2.20.2). Now I can see and get all models on run time. 
I did not make any big change. I have bound the content of the ModelVisual3D in other way like before was **SyNETAnimationViewModel3D.SyNETModel3D** and now is **SyNETAnimationViewModel3D.SyNETAnimation.SyNETAssembly**. 
So initially, I have assigned the object _SyNETModel3D_ to access SyNETAssembly object of SyNETAnimation class.
and now I have bound directly to that object.
Thank you so much for your time and gave me appropriate tips.
